test/sdk/_topology.cxx

```diff
@@ -139,6 +139,32 @@ TEST_F(Topology, ChangeState)
     EXPECT_EQ(sdk::StateEqualsTo(currentState, sdk::DeviceState::InitializingDevice), true);
 }
 
+TEST_F(Topology, MixedState)
```

Comment:
@AndreyLebedev See this test for updated usage. Is it suitable for what you want?

Reply:
Yes, thanks for the quick implementation!

Can you please make a new tag for FairMQ? And also update FairMQ version in alidist? I will then switch ODC to the newest version of FairMQ.